File: Nivel_2/Ejemplos/fibonacciIterativo.java
public class fibonacciIterativo {
    public static void main(String[] args) {
        int n = 10; // Cambia este valor para calcular otros números de Fibonacci
        System.out.println("El número Fibonacci en la posición " + n + " es: " + fibonacci(n));
    }

    public static int fibonacci(int n) {
        int result = 0;
        for (int i = n; i >= 0; i--) {result =  result + i;}
        return result;
    }
}

File: Nivel_2/Ejemplos/fibonacciIterativo.py
def fibonacci(n):
    result = 0
    for i in range(n): result = result + i
    return result
    
n = 10 #Cambia este valor para calcular otros números de Fibonacci  
print("El número Fibonacci en la posición " + str(n) + " es: " + str(fibonacci(n)))
